overlay-app/src/css/Scoreboard.css

```diff
@@ -1,82 +1,144 @@
-.scoreboard {
+.scoreboard-wrapper {
 	position: absolute;
-	top: 0px;
+	top: 0;
+	left: 0;
+	right: 0;
+}
+
+.scoreboard {
+  position: relative;
+	margin: 0 auto;
 	transition: 1s;
-	left: calc(var(--scoreboard-w) - var(--scoreboard-hw));
-	height: var(--scoreboard-h);
+  
+  
 	--scoreboard-h: calc(var(--sh) / 15);
 	--scoreboard-w: calc(var(--sw) / 2);
 	--scoreboard-hw: calc(var(--scoreboard-w) / 2);
-  	width: var(--scoreboard-w);
+	--name-h: calc(var(--scoreboard-h) * 0.9);
+  height: var(--scoreboard-h);
 	opacity: 0.99;
+	--logo-size: calc(var(--name-h) + var(--tally-h));
+  
+  --center-w: calc(var(--sw) / 10);
+  --center-hw: calc(var(--center-w) / 2);
+  --score-w: calc(var(--sw) * .04);
+	--tally-h: calc(var(--sw) * .0078);
+
+  width: var(--scoreboard-w);
+}
+
+.scoreboard.has-logos {
+	--scoreboard-w: calc(var(--sw) / 1.7);
 }
-.scoreboard .left{
+
+.scoreboard .left .logo {
+	left: 0;
+  border-radius: 0 0 0 var(--box-radius);
+}
+
+.scoreboard .right .logo {
+	right: 0;
+  border-radius: 0 0 var(--box-radius) 0;
+}
+
+.scoreboard .logo {
+  z-index: 2;
 	position: absolute;
-	top: 0px;
+	top: var(--offset);
+	height: var(--logo-size);
+	width: var(--logo-size);
+	background-color: rgba(0,0,0,.99);
+  top: calc(var(--scoreboard-h) - var(--name-h));
+}
+
+.scoreboard .logo .logo-inner {
+  --logo-padding: 5px;
+	position: absolute;
+	left: var(--logo-padding);
+	right: var(--logo-padding);
+	top: var(--logo-padding);
+	bottom: var(--logo-padding);
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-size: contain;
+}
+
+.scoreboard .side {
+  float: left;
+  position: relative;
+  width: calc(var(--scoreboard-hw) - calc(var(--center-hw)));
+  height: var(--scoreboard-h);
+}
```

Comment:
Moved some of the CSS for shared things to a generic class, specifying exceptions below